README.md

```diff
@@ -23,6 +23,14 @@ $ npm install --global git-delete-squashed
 $ git-delete-squashed
 ```
 
+You can specify an alternate branch to check for squashed merges, as well. This is useful for different names of trunk branches like `main` or `develop`.
```

Comment:
Thanks for the pull request! Can you also update the bash script in the readme so that it also accepts an argument? It would be nice to keep the two versions of the script in sync.

Reply:
(To clarify, I meant the script on line 14 of the readme, which currently hardcodes `master` rather than accepting an argument.)

Reply:
Ok, that will require a fundamental change in how the one-liner is called. Are you okay with that?

Reply:
@not-an-aardvark Just updated the readme.